index.html

```diff
@@ -652,14 +655,17 @@ <h3>
           fails, set both <a data-link-for=
           "PerformanceNavigationTiming">unloadEventStart</a> and
           <a data-link-for="PerformanceNavigationTiming">unloadEventEnd</a> to
-          0 then go to <a href="#fetch-start-step">fetch-start-step</a>.
+          0 then go to <a href="#worker-start-step">worker-start-step</a>.
           Otherwise, record <a data-link-for=
           "PerformanceNavigationTiming">unloadEventStart</a> as the time
           immediately before the unload event.
           </li>
           <li>Immediately after the unload event is completed, record the
           current time as <a data-link-for=
-          "PerformanceNavigationTiming">unloadEventEnd</a>. If the navigation
+          "PerformanceNavigationTiming">unloadEventEnd</a>.
+          </li>
+          <li id="worker-start-step">
```

Comment:
Can we label it visibly, similar to the below fetch-start-step?

Reply:
Added!